vnext/Shared/Modules/PlatformConstantsModule.cpp

```diff
@@ -10,16 +10,16 @@ using namespace winrt::Windows::Foundation::Metadata;
 
 using Method = facebook::xplat::module::CxxModule::Method;
 
-#ifndef RNW_PKG_VERSION_MAJOR
-#define RNW_PKG_VERSION_MAJOR 1000
+#ifndef RNW_MAJOR
+#define RNW_MAJOR 1000
```

Comment:
can you clarify how this works? the code below will always use these dummy values AFAICT

Reply:
Yeah, this was here before, but I think previously we only passed preprocessor definitions to resource compiler, instead of CLCompile as well. This PR adds to to MSRN CLCompile ItemMetadata, and changed names to be the same, to try to fix and reduce future confusion.

Reply:
as far as I can tell, MSVC will issue a "warning: macro redefinition", and use the value given in the `#define`, not the value passed in by /D: https://godbolt.org/z/4c45Erf5r

Reply:
I might be reading this wrong, but isn't the `#ifndef` guarding that?

We can remove this whole section though if we do not want it to locally be able to fallback. Having a single source of truth probably makes things less confusing.

Reply:
yes my bad I read it as ifdef. need more sleep.